docs/core/install/macos.md

```diff
@@ -201,7 +201,7 @@ Visual Studio for Mac installs the .NET SDK when the **.NET** workload is select
 
 | .NET SDK version      | Visual Studio version                                |
 | --------------------- | ---------------------------------------------------- |
-| 8.0                   | Visual Studio 2022 for Mac 17.8 or higher.           |
+| 8.0                   | Visual Studio 2022 for Mac 17.6 or higher.           |
```

Comment:
Hello @gewarren 

Suggest to indicate supporting .NET SDK 8.0 is a Preview Feature only in Visual Studio 2022 for mac 17.6 only.

<img width="809" alt="image" src="https://github.com/dotnet/docs/assets/62630498/f33087e2-9d29-4f1e-9352-2230250e627d">

Thanks